base/multidimensional.jl

```diff
@@ -1156,7 +1156,7 @@ the other elements are left untouched.
 ```jldoctest
 julia> x = [1., 0., 3., 0., 5.];
 
-julia> y = zeros(7);
+julia> y = Vector{Float64}(uninitialized, 7);
```

Comment:
The example is meant to show the last two zeros being preserved, so filling with uninitialized doesn't seem right.